conda.recipe/meta.yaml

```diff
@@ -5,16 +5,23 @@ package:
 source:
   path: ../
 
+build:
+  number: {{ environ.get('GIT_DESCRIBE_NUMBER', 0) }}
+  {% if environ.get('GIT_DESCRIBE_NUMBER', '0') == '0' %}string: py{{ environ.get('PY_VER').replace('.', '') }}_0
+  {% else %}string: py{{ environ.get('PY_VER').replace('.', '') }}_{{ environ.get('GIT_BUILD_STR', 'GIT_STUB') }}{% endif %}
+
```

Comment:
That's the one! It throws a nasty error, and i don't know what else it
breaks. Suffice to say, that's why I am going after a non-kernel notebook
first, which is almost working...

On Mon, Nov 16, 2015 at 5:08 PM Damián Avila <notifications@github.com>
wrote:

> In conda.recipe/meta.yaml
> <https://github.com/ContinuumIO/nbpresent/pull/13#discussion_r44991484>:
>
> > @@ -5,16 +5,23 @@ package:
> >  source:
> >    path: ../
> >
> > +build:
> > +  number: {{ environ.get('GIT_DESCRIBE_NUMBER', 0) }}
> > +  {% if environ.get('GIT_DESCRIBE_NUMBER', '0') == '0' %}string: py{{ environ.get('PY_VER').replace('.', '') }}_0
> > +  {% else %}string: py{{ environ.get('PY_VER').replace('.', '') }}_{{ environ.get('GIT_BUILD_STR', 'GIT_STUB') }}{% endif %}
> > +
>
> OK, I found this:
> https://github.com/jupyter/notebook/blob/master/notebook/base/zmqhandlers.py#L90,
> are you taking about this???
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ContinuumIO/nbpresent/pull/13/files#r44991484>.
>